src/cache/cache_las.c

```diff
@@ -1146,6 +1146,7 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
     saved_btree_id = 0;
     saved_timestamp = WT_TS_NONE;
     saved_txnid = WT_TXN_NONE;
+    special_ondisk_value = false;
```

Comment:
"Special" isn't very descriptive in my opinion. Maybe `special_ondisk_value` => `globally_visible_ondisk_value`? If that is accurate.

Reply:
Nice, changed.